### PR TITLE
Unification code

### PR DESCRIPTION
**Pull request by griff-pyrofex**
Chris has already reviewed this code in a different directory structure

### CODE REVIEW

File: storage/src/main/scala/coop/rchain/storage/Key.scala [hunk 1]
```diff
@@ -0,0 +1,147 @@
+/*               __          __                                         *\
+**    __________/ /_  ____ _/_/___                                      **
+**   / ___/ ___/ __ \/ __ `/ / __ \     RChain API                      **
+**  / /  / /__/ / / / /_/ / / / / /     (c) http://rchain.coop          **
+** /_/   \___/_/ /_/\____/_/_/ /_/                                      **
+\*                                                                      */
+
+package coop.rchain.Storage
```
Comment (kirkwood) [inline storage/src/main/scala/coop/rchain/storage/Key.scala]: Suggest downcasing that "S".

**Comment by griff-pyrofex** [comment]
Yes, correct directory structure.

**Comment by griff-pyrofex** [comment]
This is the code that does query unification.

**Comment by griff-pyrofex** [comment]
This PR was supplanted by PR #39

**Pull request closed without merging**